infra/base-images/base-runner/parse_options

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+"""Helper script for parsing custom fuzzing options."""
+import configparser
+import os
+import sys
+
+if len(sys.argv) < 3:
```

Comment:
nit: why not make this end in .py?

Also why not put this in a main function?

Reply:
was thinking about consistency with the other scripts, but they seem to be all bash and the only other python one is `.py`, so renamed this one too

didn't have the main function since it's a tiny helper script, but added it now